environments/custom/playbook-lvm.yml

```diff
@@ -1,5 +1,14 @@
+# https://docs.debops.org/en/stable-3.1/ansible/roles/lvm/defaults/main.html#volume-groups-and-logical-volumes
 - name: Manage LVM volumes
   hosts: generic
+  gather_facts: yes
```

Comment:
Do not mix yes + true. Use true to make yamllint happy.

Reply:
done